PowerArubaCX/Public/Firmware.ps1

```diff
@@ -0,0 +1,37 @@
+﻿function Get-ArubaCXfirmware {
+
+    <#
+      .SYNOPSIS
+      Get Aruba CX firmware
+
+      .DESCRIPTION
+      Get all informations about Aruba CX firmware
+
+      .EXAMPLE
+      Get-ArubaCXfirmware
+
+        current_version   : Virtual.10.04.0001
```

Comment:
No need to make a example of display but a description of function (with parameter when it is possible)